static/app/views/issueDetails/groupSidebar.tsx

```diff
@@ -0,0 +1,276 @@
+import {Fragment, useCallback, useEffect, useState} from 'react';
```

Comment:
Looks like you didn't delete the other one

Reply:
@evanpurkhiser fixed